homeassistant/components/generic/strings.json

```diff
@@ -13,8 +13,7 @@
       "stream_no_route_to_host": "Could not find host while trying to connect to stream",
       "stream_io_error": "Input/Output error while trying to connect to stream. Wrong RTSP transport protocol?",
       "stream_unauthorised": "Authorisation failed while trying to connect to stream",
```

Comment:
I think quite a lot of these can be deleted now that there is no way to generate them.

Reply:
I'll remove `stream_unauthorised` as that seems to be gone.
`stream_not_permitted`, `stream_no_route_to_host`, and `stream_io_error` were left in because they are still in the code. It seems like the stream worker won't catch those errors, but I didn't test them - can you help check if they are still valid?
Any others I've missed?